src/test/java/RenameAPlaylist.java

```diff
@@ -1,53 +1,66 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.Keys;
 
 public class RenameAPlaylist extends BaseTest {
 
-    String email = "holostenco.yuliya@gmail.com";
-    String Password = "te$t$tudent";
-    String xpath = "//ul//li[@class='playlist playlist']";
-    String css ="li>input";
-   String  newPlaylistName=" -Best Club music Selection";
+
+    By newPlaylistLocator = By.cssSelector(".playlist:nth-child(3)");
+    By getNewPlaylistTextFieldLocator =By.cssSelector("input[name='name']");
+    String  newPlaylistName= "Best";
 
     @Test
-    public void renameSelectedPlaylist(){
+
+    public void renameSelectedPlaylist() {
         login();
         chooseAPlaylist();
+        nameNewPlaylist();
+        Assert.assertEquals(newPlaylistName, getTextNameNewPlaylist());
 
-wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
-        WebElement renamedPlaylist=driver.findElement(By.cssSelector(css));
-        renamedPlaylist.sendKeys(newPlaylistName+"\n");
-        Assert.assertTrue(renamedPlaylist.isDisplayed());
     }
+    public void login() {
+        WebElement emailField = driver.findElement(By.xpath("//input[@placeholder='Email Address']"));
+        waitForElementToBeClickable(emailField).click();
+        emailField.sendKeys(email);
+
+        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));
+        waitForElementToBeClickable(passwordField).click();
+        passwordField.sendKeys(Password);
+
+        WebElement submitButton =driver.findElement(By.xpath("//button[@type='submit']"));
+        waitForVisibilityOfElement(submitButton).click();
 
+
+    }
     private void chooseAPlaylist() {
-        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(xpath)));
-        WebElement playlist = driver.findElement(By.xpath(xpath));
+        WebElement playlist = driver.findElement(newPlaylistLocator);
+        waitForVisibilityOfElement(playlist);
+        Actions actions=new Actions(driver);
         actions.doubleClick(playlist).perform();
     }
+    private void nameNewPlaylist() {
```

Comment:
I recommend to use in the functions naming: verb, long names and follow code convention

For example `renamePlayList`

![image](https://user-images.githubusercontent.com/104761482/200170663-7c5442a7-ac74-469d-87cb-ab045f6f4df7.png)